flink-connectors/flink-connector-aws-kinesis-data-streams/src/test/java/org/apache/flink/streaming/connectors/kinesis/testutils/KinesaliteContainer.java

```diff
@@ -45,23 +56,18 @@
 public class KinesaliteContainer extends GenericContainer<KinesaliteContainer> {
     private static final String ACCESS_KEY = "access key";
     private static final String SECRET_KEY = "secret key";
+    private static final Region REGION = Region.US_EAST_1;
 
     public KinesaliteContainer(DockerImageName imageName) {
         super(imageName);
 
+        System.setProperty("com.amazonaws.sdk.disableCertChecking", "true");
```

Comment:
Is this still needed although we are setting the property through maven?

Reply:
You're right! I thought I would need to retain this for running the tests outside maven, but works.